modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -90,6 +91,107 @@ output:
+
+=== Scenario: Data and table schema match (schema evolution turned on or off) 
+
+An output message matches the existing table schema and the `schema_evolution.enabled` field is set to `true` or `false`.
+
+The target Snowflake table has two columns:
+
+- `product_id` (NUMBER)
+- `product_code` (STRING)
+
+A pipeline generates the following message:
+
+```json
+{"product_id": 521, "product_code": “EST-PR”}
+```
+
+In this scenario:
+
+- The JSON keys in the message (`"product_id"` and `"product_code"`) match column names in the target Snowflake table.
+- The message values match the column data types. (If there was a data mismatch, the message would be rejected.)
+- Redpanda Connect inserts the message values into a new row in the target Snowflake table.
++
+|===
+| product_id | product_code
+
+^| 521
+^| EST-PR
+|===
+ 
+=== Scenario: Data and table schema mismatch (schema evolution turned on) 
+
+An output message includes schema updates and the `schema_evolution.enabled` field is set to `true`.
+
+The target Snowflake table has the same two columns as for the <<scenario-data-and-table-schema-match-schema-evolution-turned-on-or-off, previous scenario>>:
+
+- `product_id` (NUMBER)
+- `product_code` (STRING)
+
+This time the pipeline generates the following message:
+
+```json
+{"product_batch": 11111, "product_color": “yellow”}
+```
+
+In this scenario:
+
+- The JSON keys (`"product_batch"` and `"product_color"`) do not match column names in the target Snowflake table.
+- As schema evolution is enabled, Redpanda Connect adds two new columns to the target table with data types derived from the output message values. For more information about the mapping of data types, see <<supported-data-formats-for-snowflake-columns, Supported data formats for Snowflake columns>>  
+- Redpanda Connect inserts the message values into a new table row.
++
+|===
+| product_id | product_code | product_batch | product_color
+
+^| (null)
+^| (null)
+^| 11111
+^| yellow
+
+|===
++
+NOTE: You can <<schema_evolution-processors,configure processors>> to override the schema updates derived from the message values.
+
+=== Scenario: Data and table schema mismatch (schema evolution turned off)
+
+An output message includes schema updates and the `schema_evolution.enabled` field is set to `false`.
+
+The target Snowflake table has the same two columns:
+
+- `product_id` (NUMBER)
+- `product_code` (STRING)
+
+The pipeline generates the same message as for the <<scenario-data-and-table-schema-mismatch-schema-evolution-turned-on,previous scenario>>:
+
+```json
+{"product_batch": 11111, "product_color": “yellow”}
+```
+
+In this scenario:
+
+- The JSON keys (`"product_batch"` and `"product_color"`) do not match any existing column names.
+- As schema evolution is turned off, Redpanda Connect ignores the extra column names and values and inserts a row of null values.
```

Comment:
```suggestion
- Because schema evolution is turned off, Redpanda Connect ignores the extra column names and values and inserts a row of null values.
```